[0m[[0minfo[0m] [0mRunning GeoPicassoRx [0m
[0m[[0mdebug[0m] [0mWaiting for threads to exit or System.exit to be called.[0m
[0m[[0mdebug[0m] [0m  Classpath:[0m
[0m[[0mdebug[0m] [0m	/home/ben/Programming/ScalaGeneralWorkspace/CirclesinApril/target/scala-2.11/classes[0m
[0m[[0mdebug[0m] [0m	/home/ben/Programming/ScalaGeneralWorkspace/CirclesinApril/lib/rxscala_2.11-0.24.1.jar[0m
[0m[[0mdebug[0m] [0m	/home/ben/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.11.6.jar[0m
[0m[[0mdebug[0m] [0m	/home/ben/.ivy2/cache/org.jsoup/jsoup/jars/jsoup-1.7.2.jar[0m
[0m[[0mdebug[0m] [0m	/home/ben/.ivy2/cache/io.reactivex/rxscala_2.11/jars/rxscala_2.11-0.24.1.jar[0m
[0m[[0mdebug[0m] [0m	/home/ben/.ivy2/cache/io.reactivex/rxjava/jars/rxjava-1.0.8.jar[0m
[0m[[0mdebug[0m] [0mWaiting for thread run-main-31 to terminate.[0m[0m[[0mdebug[0m] [0m	Thread run-main-0 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mInterrupting thread RxComputationThreadPool-2[0m
[0m[[0mdebug[0m] [0m	Interrupted RxComputationThreadPool-2[0m
[0m[[0mdebug[0m] [0mInterrupting thread RxComputationThreadPool-1[0m
[0m[[0mdebug[0m] [0m	Interrupted RxComputationThreadPool-1[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
[0m[[0mdebug[0m] [0mExited with code 0[0m
0)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable$1.call(Observable.java:144)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable$1.call(Observable.java:144)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable.subscribe(Observable.java:7585)
	at rx.Observable.subscribe(Observable.java:7271)
	at rx.lang.scala.Observable$class.subscribe(Observable.scala:133)
	at rx.lang.scala.JavaConversions$$anon$2.subscribe(JavaConversions.scala:53)
	at GeoPicassoRx.<init>(GeoPicassoRx.scala:42)
	at GeoPicassoRx$.main(GeoPicassoRx.scala:189)
	at GeoPicassoRx.main(GeoPicassoRx.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sbt.Run.invokeMain(Run.scala:72)
	at sbt.Run.run0(Run.scala:65)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:54)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Logger$$anon$4.apply(Logger.scala:90)
	at sbt.TrapExit$App.run(TrapExit.scala:244)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IndexOutOfBoundsException: 0
	at scala.collection.LinearSeqOptimized$class.apply(LinearSeqOptimized.scala:65)
	at scala.collection.immutable.List.apply(List.scala:84)
	at GeoPicassoRx.createDoc(GeoPicassoRx.scala:175)
	at GeoPicassoRx$$anonfun$1.apply$mcV$sp(GeoPicassoRx.scala:39)
	at rx.lang.scala.ImplicitFunctionConversions$$anon$3.call(ImplicitFunctionConversions.scala:64)
	at rx.internal.operators.OperatorDoOnSubscribe.call(OperatorDoOnSubscribe.java:39)
	at rx.internal.operators.OperatorDoOnSubscribe.call(OperatorDoOnSubscribe.java:26)
	at rx.Observable$1.call(Observable.java:140)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable$1.call(Observable.java:144)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable$1.call(Observable.java:144)
	at rx.Observable$1.call(Observable.java:136)
	at rx.Observable.subscribe(Observable.java:7585)
	at rx.Observable.subscribe(Observable.java:7271)
	at rx.lang.scala.Observable$class.subscribe(Observable.scala:133)
	at rx.lang.scala.JavaConversions$$anon$2.subscribe(JavaConversions.scala:53)
	at GeoPicassoRx.<init>(GeoPicassoRx.scala:42)
	at GeoPicassoRx$.main(GeoPicassoRx.scala:189)
	at GeoPicassoRx.main(GeoPicassoRx.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at sbt.Run.invokeMain(Run.scala:72)
	at sbt.Run.run0(Run.scala:65)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:54)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:58)
	at sbt.Logger$$anon$4.apply(Logger.scala:90)
	at sbt.TrapExit$App.run(TrapExit.scala:244)
	at java.lang.Thread.run(Thread.java:745)
[0m[[0mdebug[0m] [0m	Thread run-main-0 exited.[0m
[0m[[0mdebug[0m] [0mInterrupting remaining threads (should be all daemons).[0m
[0m[[0mdebug[0m] [0mSandboxed run complete..[0m
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1708)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:1708)
	at scala.Option.foreach(Option.scala:236)
	at sbt.BuildCommon$class.toError(Defaults.scala:1708)
	at sbt.Defaults$.toError(Defaults.scala:35)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$34$$anonfun$apply$35.apply(Defaults.scala:678)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$34$$anonfun$apply$35.apply(Defaults.scala:676)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
	at sbt.std.Transform$$anon$4.work(System.scala:64)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:244)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
[0m[[31merror[0m] [0m(compile:[31mrunMain[0m) Nonzero exit code: 1[0m
